shaders/Boids_ComputeFX.sdsl

```diff
@@ -143,15 +143,12 @@ shader Boids_ComputeFX : ComputeShaderBase, BoidsData, Global //Time and Timeste
                                     float4(zDir, 0),
                                     float4(0, 0, 0, 1));
 
-
         Matrix m;
         m.m = rotation;
-        m.m._41_42_43 = position;

```

Comment:
Thanks for the fix commit! However, it seems that even the necessary translation element have been removed.

Reply:
Oh yes, I deleted too much... should have tested it better. Thanks!